utils/build_swift/README.md

```diff
@@ -8,5 +8,5 @@ the Swift build-script.
 You may run the unit test suite using the command:
 
 ```sh
-$ python utils/build_swift/run_tests.py
+$ utils/build_swift/run_tests.py
```

Comment:
This isn't entirely portable.  I'd leave that - e.g. shebangs don't apply on Windows.

Reply:
OK, I'll change it to `$ python3 utils/build_swift/run_tests.py`